### ISSUE TITLE
bz close with inline comment does not work

### ISSUE THREAD

**Issue by feld** [pgollucci/freebsd-bugzilla-cli#4]
```
> bz help close
Usage: bz close [-c comment | -e] pr
       bz take -h

Options:
    -c    -- optional comment (default none)
    -e    -- optional spawn vi
    -h    -- this help message

Args:
    pr    -- pr number

Will mark pr as Closed, FIXED and by default no comment.
```
Ok, here goes nothing:

```
> bz close -c 'Committed with minor changes, thanks!' 199343
```

-- nothing happens --

Maybe the order is documented wrong?

```
> bz close 199343 -c 'Committed with minor changes, thanks!'
 * Using https://bugs.freebsd.org/bugzilla/xmlrpc.cgi
 * Logging in
 * Logging in
 * Modified the following fields in bug 199343
 * resolution  : removed
 * resolution  : added FIXED
 * status      : removed New
 * status      : added Closed
```

No comment is added

**Comment by pgollucci** [comment]
Try the 0.10.6 in the tree now I specifically fixed some stuff with close.

On Wed, Aug 12, 2015 at 12:55 PM, Mark Felder <notifications@github.com>
wrote:

> > bz help close
> Usage: bz close [-c comment | -e] pr
>        bz take -h
>
> Options:
>     -c    -- optional comment (default none)
>     -e    -- optional spawn vi
>     -h    -- this help message
>
> Args:
>     pr    -- pr number
>
> Will mark pr as Closed, FIXED and by default no comment.
>
> Ok, here goes nothing:
>
> > bz close -c 'Committed with minor changes, thanks!' 199343
>
> -- nothing happens --
>
> Maybe the order is documented wrong?
>
> > bz close 199343 -c 'Committed with minor changes, thanks!'
>  * Using https://bugs.freebsd.org/bugzilla/xmlrpc.cgi
>  * Logging in
>  * Logging in
>  * Modified the following fields in bug 199343
>  * resolution  : removed
>  * resolution  : added FIXED
>  * status      : removed New
>  * status      : added Closed
>
> No comment is added
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pgollucci/freebsd-bugzilla-cli/issues/4>.
>



-- 
---------------------------------------------------------------------------------
Curb: Your ride is here
4096R/D21D2752
<http://pgp.mit.edu/pks/lookup?op=get&search=0xF699A450D21D2752> ECDF B597
B54B 7F92 753E  E0EA F699 A450 D21D 2752
Philip M. Gollucci (pgollucci@p6m7g8.com) c: 703.336.9354
Member,                           Apache Software Foundation
Committer,                        FreeBSD Foundation
Consultant,                       P6M7G8 Inc.
Sr. Director IT Operations,       Curb

What doesn't kill us can only make us stronger;
Except it almost kills you.

**Comment by pgollucci** [comment]
I think I'll add some shortcut flags too, "Committed", "Committed with minor changes", and "massive changes".

**Comment by pgollucci** [comment]
fixed in 0.10.8
